### DISCUSSION TITLE
Export/copy results from profiler extension?

### DISCUSSION THREAD

**Discussion by HWidforss** [microsoft/azuredatastudio#21454] [General]
Is there an easy way to export or copy all of the results from the profiler extension? It is possible to copy the contents of one cell, but I haven't found a way with multiple cells.
The use case is that I want to copy the results (CPU, Reads, Writes, Duration) into f.e. Excel where I can calculate averages and the like from multiple executions of a query.

**Comment by cheenamalhotra** [comment]
Not as of now I think, open feature request: https://github.com/microsoft/azuredatastudio/issues/12987
